lucene/core/src/java/org/apache/lucene/util/bkd/BKDRadixSelector.java

```diff
@@ -82,30 +82,40 @@ public BKDRadixSelector(int numDim, int bytesPerDim, int maxPointsSortInHeap, Di
    * the split happens. The method destroys the original writer.
    *
    */
-  public byte[] select(PointWriter points, PointWriter left, PointWriter right, long from, long to, long partitionPoint, int dim) throws IOException {
+  public byte[] select(PathSlice points, PathSlice[] slices, long from, long to, long partitionPoint, int dim, int dimCommonPrefix) throws IOException {
```

Comment:
can you document in javadocs that `slices` should have a length of 2 and how it gets filled?

Reply:
I have relaxed the condition and it should be bigger than 1. Javadocs added